test/blackbox-tests/test-cases/melange/empty-entries-target-dir-exists.t

```diff
@@ -18,4 +18,3 @@ Create the dist folder
 
   $ dune build @melange-dist
   $ ls _build/default/dist
-  melange.js
```

Comment:
not sure what happened here yet

Reply:
Okay, I understand. The sources are generated in the object directories so they aren't included here.

Reply:
Can you explain this in other words?

Reply:
It just means that melange.js exists in `_build/default/dist/.dist.mobjs/` rather than `_build/default/dist`. Since it's a generated module, that seems like the better behavior as long as it doesn't break anything.